### PR TITLE
Fixing Public Jira Issue : APIMANAGER-3934

### PR DESCRIPTION
**Pull request by ChamNDeSilva**
Fix Viewing APIs which are in created state in API Store

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java [hunk 1]
```diff
@@ -2059,115 +2059,121 @@ public static NativeArray jsFunction_getAPI(Context cx, Scriptable thisObj,
...
-                    if (apiOwner == null) {
-                        apiOwner = APIUtil.replaceEmailDomainBack(apiIdentifier.getProviderName());
-                    }
-                    row.put("apiOwner", row, apiOwner);
-                    row.put("isAdvertiseOnly", row, api.isAdvertiseOnly());
-                    row.put("redirectURL", row, api.getRedirectURL());
+                        row.put("subscriptionAvailability", row, api.getSubscriptionAvailability());
+                        row.put("subscriptionAvailableTenants", row, api.getSubscriptionAvailableTenants());
+                        row.put("isDefaultVersion",row,api.isDefaultVersion());
+                        row.put("transports",row,api.getTransports());
```
Comment (harsha89) [inline components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java]: Two Small formatting issue. Need to have space after ,

File: components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java [hunk 2]
```diff
@@ -2059,115 +2059,121 @@ public static NativeArray jsFunction_getAPI(Context cx, Scriptable thisObj,
...
+                        row.put("techOwner", row, api.getTechnicalOwner());
+                        row.put("techOwnerMail", row, api.getTechnicalOwnerEmail());
+                        row.put("visibility", row, api.getVisibility());
+                        row.put("visibleRoles", row, api.getVisibleRoles());
+
+                        Set<URITemplate> uriTemplates = api.getUriTemplates();
+                        List<NativeArray> uriTemplatesArr = new ArrayList<NativeArray>();
+                        if (uriTemplates.size() != 0) {
+                            NativeArray uriTempArr = new NativeArray(uriTemplates.size());
+                            Iterator i = uriTemplates.iterator();
```
Comment (harsha89) [inline components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java]: use meaningful variable names

Comment (ChamNDeSilva) [inline components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java]: Fixed in PR https://github.com/wso2/carbon-apimgt/pull/1093

File: components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java (hunk 1 above)
Comment (ChamNDeSilva) [inline components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java]: Fixed in PR https://github.com/wso2/carbon-apimgt/pull/1093

**Pull request merged**